lib/imap_processor.rb

```diff
@@ -21,7 +21,9 @@ def process
     sitename = AppSettings["settings.site_name"]
     message =  get_content_from_mail
     raw = raw_body_from_mail.nil? ? "" : raw_body_from_mail
-    cc = @email.cc.join(",")
+    if !@email.cc.nil?
```

Comment:
The `!` and `nil?` double negative I would discourage. Since this is for a Rails app it would be better to do:

```
if @email.cc.present?
  cc = @email.cc.join(",")
end
```

With the safe navigation operator the even more idiomatic Ruby would be:

```
cc = @email.cc&.join ','
```